dbt/tests/generic/test_row_values_match_after_join.sql

```diff
@@ -28,8 +28,8 @@
 --
 -- Optional parameters:
 --
---    * join_type(str): The type of join to use, e.g. "inner join" or
---    "left join". Defaults to "inner join".
+--    * join_type(str): The type of join to use, e.g. "inner" or "left".
+--    Defaults to "inner".
```

Comment:
I tweaked this test to match the interface used by `res_class_matches_join`, since on second thought it feels more intuitive for me for "join type" to refer to the type of join ("inner" or "outer") rather than the full join string itself ("inner join" or "outer join"). I'm definitely open to pushback if you disagree. There's no need to adjust any tests to use this new interface, since `join_type` isn't actually specified in any active tests right now.